### PR TITLE
Simplify ArticlesController#feed

### PR DESCRIPTION
**Pull request by rhymes**
There's unneeded complexity in the logic of the feed method, this way what separates each path should be clearer.

## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Documentation Update

## Description

I came across ArticlesController#feed and noticed that the ORM logic was a bit convoluted and repeated in each if/else clause. Since AR is a lazy and composable ORM, I decided to simplify the logic.

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

### CODE REVIEW

File: app/controllers/articles_controller.rb [hunk 1]
```diff
@@ -8,32 +8,28 @@ class ArticlesController < ApplicationController
...
-          order("published_at DESC").
-          page(@page).per(12)
+        @articles.where(user_id: @user.id)
       elsif @user = Organization.find_by_slug(params[:username])
-        @articles = Article.where(published: true, organization_id: @user.id).
-          includes(:user).
-          select(:published_at, :processed_html, :user_id, :organization_id, :title, :path).
-          order("published_at DESC").
-          page(@page).per(12)
+        @articles.where(user_id: @organization.id).includes(:user)
```
Comment (maestromac) [inline app/controllers/articles_controller.rb]: Based on the failing spec, I think this is the change you need.
```suggestion
        @articles.where(organization_id: @user.id).includes(:user)
```

Comment (rhymes) [reply app/controllers/articles_controller.rb]: Oh, shoot! I should stop coding at night :D

**Comment by rhymes** [comment]
Thanks @maestromac!

**Comment by lightalloy** [comment]
Nice refactoring )

**Pull request merged**